Lesson30/Firat/my-app/src/components/Exercise4/index.jsx

```diff
@@ -0,0 +1,18 @@
+import React, { useState } from "react";
```

Comment:
No need to import React
```suggestion
import { useState } from "react";
```